server/sockets.js

```diff
@@ -2,8 +2,8 @@ var iolib = require('socket.io')
 	, log = require("./log.js").log
 	, BoardData = require("./boardData.js").BoardData;
 
-var MAX_EMIT_COUNT = 64; // Maximum number of draw operations before getting banned
-var MAX_EMIT_COUNT_PERIOD = 5000; // Duration (in ms) after which the emit count is reset
+var MAX_EMIT_COUNT = 200; // Maximum number of draw operations before getting banned
+var MAX_EMIT_COUNT_PERIOD = 1000; // Duration (in ms) after which the emit count is reset
```

Comment:
I think this is not related to this PR..

Reply:
This PR increases the polling rate of the pencil tool leading to a much smoother experience but also an increased message count. I just did some testing and found that I can't exceed 200 messages over 5 seconds unless I actively try. Even attempting to fill out a larger area by rapidly whacking the pencil left and right tends to not exceed that number.